src/sass/table/table.scss

```diff
@@ -131,6 +163,11 @@ $density-relaxed-img-radius: 8px;
     text-align: center;
     width: 48px;
 }
+/* Not sure of downstream dev consumers' ability to utilize this.
+   Convert to long-hand if problematic. */
+.table--mode-selection tr:has(input:checked) :where(td, th) {
```

Comment:
Did you include this by accident? Looks unrelated to the PR.

Also, `:has` only has 91.66% global compatibility: https://caniuse.com/css-has

Reply:
It's not an accident. This is to highlight the row on selection. Are you pulling down PRs locally when you review?

This doesn't impact function and is purely decorative. We do use `:has` for similar types of progressive enhancements, for which current level of support is not really an issue.

Reply:
The markup was already done for selection. This is just a styling change; not all of those are specified granularly in PRs. No worries. Better safe than sorry. 😄